packages/vue/src/types/task.ts

```diff
@@ -26,7 +26,7 @@ export interface Task {
   company: string,
   priority: TaskPriority,
   startDate: Date,
-  dueDate: Date,
+  dueDate: Date | null,
```

Comment:
Why do we add null? Looks like it is not correct type for value of dateBox https://github.com/DevExpress/DevExtreme/blob/22_2/js/ui/date_box.d.ts#L217